tests/src/test/java/io/streamnative/pulsar/handlers/kop/KafkaRequestHandlerTest.java

```diff
@@ -715,7 +715,8 @@ public void testMetadataForNonPartitionedTopic(short version) throws Exception {
         final MetadataRequest request = new MetadataRequest(Collections.singletonList(topic), false, version);
         final CompletableFuture<AbstractResponse> responseFuture = new CompletableFuture<>();
         handler.handleTopicMetadataRequest(
-                new KafkaHeaderAndRequest(header, request, PulsarByteBufAllocator.DEFAULT.heapBuffer(), null),
+                new KafkaHeaderAndRequest(header,
+                        request, PulsarByteBufAllocator.DEFAULT.heapBuffer(), null),
                 responseFuture);
```

Comment:
Could you revert all unrelated changes?

Reply:
In this file. Only some indent changes were made. It's confused and breaks the `git blame` history.

Reply:
done. all unrelated changes have been reverted. PTAL again.